src/nginx/conf/nginx-config.template

```diff
@@ -522,6 +534,12 @@ stream {
     access_log /var/lib/nginx/log/stream_access.log stream;
           {{ end }}
     proxy_pass be_stream_{{$cfg.Namespace}}_{{$cfg.Name}};
+
+    {{ if isTrue (env "ROUTER_ENABLE_UNSAFE_ANNOTATIONS") }}
```

Comment:
the indentation of the template parts  (`{{ .. }}`) except `{{ $value }}` is off.
Could you possibly indent `{{ if isTrue (env "ROUTER_ENABLE_UNSAFE_ANNOTATIONS") }}` at the same level as `{{ end }}` in line 535 and the other lines accordingly. Same applies for the other template.

apologies. I didn't catch it during the first review :(

Reply:
adjusted in baf97e62e598799c5d5ab59a9660773946e18a48